feeder/feeder.py

```diff
@@ -18,12 +18,15 @@
 # Set up error handling
 import opbeat
 from opbeat.handlers.logging import OpbeatHandler
-if 'OPBEAT_SECRET' in os.environ:
+secret_token= os.environ.get('OPBEAT_SECRET')
```

Comment:
There's many secret tokens (opbeat, entsoe etc..), so I'd prefer to keep the old `if` clause, which checked for the existence of `OPBEAT_SECRET`.